app/javascript/styles/mastodon/components.scss

```diff
@@ -6645,6 +6645,23 @@ noscript {
       margin-bottom: 5px;
     }
 
+    &__show {
+      font-size: 14px;
+      font-weight: 500;
+      cursor: pointer;
+      padding: 10px;
+      margin: 0 -10px;
+    }
+
+    &__empty {
+      font-size: 14px;
+      font-weight: 500;
+      cursor: pointer;
+      padding: 10px;
+      margin: 0 -10px;
+      color: $darker-text-color;
+    }
```

Comment:
You probably don't want to duplicate styles here, instead I'd suggest to use some base class (say `&__content`) with current styles from `&__show`, and nested in `&-empty`.

```sass
&__content {
   ...

  &-empty {
    color: ...;
  }
}
```

So in the range [@ThibG commented on](https://github.com/tootsuite/mastodon/pull/14269/files#r451766982) you can use the same `div` (independent of `value`), but give it proper `className` using [`classnames` library](https://www.npmjs.com/package/classnames) ([example usage here](https://github.com/tootsuite/mastodon/blob/1fb92037e432913902a5e5c8a5b673b036d84cb8/app/javascript/mastodon/components/status_content.js#L190-L194)).

Reply:
Yes, nested style is completely better.
And I also want to try classnames library if I can, thank you.